src/app/cart/cart-item-list/cart-item-list.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+import { Item } from '../../models/item';
+import { CartService } from '../cart.service';
+
+@Component({
+  selector: 'app-cart-item-list',
+  templateUrl: './cart-item-list.component.html',
+})
+export class CartItemListComponent implements OnInit, OnDestroy {
+  cartItemList: [Item, number][] = [];
+  private onDestroy = new Subject<void>();
+
+  constructor(private cartService: CartService) {}
+
+  ngOnInit() {
+    this.cartService.cartItemList$
```

Comment:
remember to store the subscription so we can unsubscribe and prevent memory leaks